examples/image/private_base_image.py

```diff
@@ -3,48 +3,50 @@
 
 """
 This example demonstrates how to build a new image using a private base image,
-then push the resulting image back to the same private registry.
-
-This workflow will change in the next couple weeks, but for now:
-
-First run the old CLI command to create a file. This will extract out from your local docker daemon logged in tokens.
-```
-union create imagepullsecret
-```
-
-
-Take the file that was created and then call the following to create a registry secret:
-```
-flyte create secret --type image_pull pingsutw --from-file <PATH/to/above file>
-```
 
 Keep in mind we only support one secret for now for a given image. If you have a private registry that you need to pull
 from, and a different private registry you want to push to, and hence need more than one registry secret, this is not
 supported.
 
-Note that you do not need a secret to push to (or pull from) the default remote builder registry. So in the example
-below, if you drop the `.clone(...)` call, you can also drop the `secrets` argument in the TaskEnvironment.
+Note that you do not need a secret to push to (or pull from) the default remote builder registry.
 
+Please see `flyte create secret --help` for all the options.
+"""
+
+BASE_IMG_URL = "ghcr.io/username/private:123123"
+
+"""
+In this first example, set the registry_secret on the image, so that when building the image, the remote builder
+can pull the base image from the private registry, and build from there. However since the registry that the remote
+builder pushes to is already availble for tasks, you don't need to specify a secret in the task environment itself.
 """
 image = (
-    Image.from_base("pingsutw/private:d1742efed83fc4b18c7751e53e771bbe")
-    .clone(registry="docker.io/pingsutw", name="private", registry_secret="pingsutw")
+    Image.from_base(BASE_IMG_URL)
+    .clone(name="from-private-image5", registry_secret="my_registry_secret")
```

Comment:
We also need to specify the registry here, right? To push the image to the private registry